spec/index.bs

```diff
@@ -2187,13 +2187,13 @@ dictionary GPUBindGroupLayoutEntry {
     boolean hasDynamicOffset = false;
 
     // Used for uniform buffer and storage buffer bindings.
-    GPUSize64 minBufferBindingSize;
+    GPUSize64 minBufferBindingSize = 0;
```

Comment:
I don't understand why we are adding defaults here. Could you explain it in different words, perhaps?

Reply:
Removed.